static/app/components/replays/breadcrumbs/replayTimeline.tsx

```diff
@@ -51,6 +51,13 @@ function ReplayTimeline({}: Props) {
               <MinorGridlines duration={duration} width={width} />
               <MajorGridlines duration={duration} width={width} />
               <TimelineScubber />
+              <UnderTimestamp paddingTop="52px">
```

Comment:
Why do we have 2 different paddingTop values?

Reply:
Inside of `<Stacked>` everything is effectively `position: absolute; top: 0; height: 100%;`. So what we need to do with these two is move them down from the top so sit in their row properly. 
Except it's using grid instead of absolute -> so content is not removed from the document flow. Therefore instead of setting `top: 52px;` padding or margin does work.

I guess with grid i could try some other settings, but this seemed straight forward and contained.